apps/sms/views/conversation/js/information.js

```diff
@@ -209,6 +209,12 @@ function createListWithMsgInfo(message) {
   if (message.delivery === 'received' ||
       message.delivery === 'not-downloaded') { // received message
     list.push(message.sender);
+    if (message.type === 'mms') {
+      message.deliveryInfo.forEach(function(info) {
+        list.push({number: info.receiver,
+                   infoBlock: createReportDiv(info)});
+      });
+    }
```

Comment:
@julienw I'm sorry for not testing it. I'll need to fix up my desktop environment before I proceed